etc/linux-gnome-shell/syncthing.desktop

```diff
@@ -0,0 +1,9 @@
+[Desktop Entry]
+Version=1.0
+Name=syncthing
+GenericName=syncthing
+Comment=syncthing
+Exec=/usr/bin/syncthing -no-browser -no-restart -logflags=0
+Terminal=false
+Type=Application
+X-Desktop-File-Install-Version=0.22
```

Comment:
Just out of curiosity and because a google search doesn't bring anything up: What is this option about?

Reply:
@imsodin not sure which option are you referring to, probably `X-Desktop-File-Install-Version`, honestly I don't understand it either, just found a sample at: https://unix.stackexchange.com/questions/209898/running-program-on-startup-fedora-22-2015/228376#228376 and adopted it to syncthing needs

Reply:
As its use it unknown and the file works without it, I would remove it.

Reply:
Hello, X-Desktop-File-Install-Version tag comes from "desktop-file-install" application which is under "desktop-file-utils". It is a utility to install, or optionally edit desktop files. That 0.22 is the version number which belongs to in which version of "desktop-file-install" application used. (Currently "desktop-file-utils" package version is 0.23 ) : https://www.freedesktop.org/wiki/Software/desktop-file-utils/

Reply:
Thanks for the input. From that link:
> desktop-file-install: installs a desktop file to the applications directory, optionally munging it a bit in transit.
I assume adding this line is part of the munging, which to me is another reason to not have this line in the distributed desktop file. If the user then installs the desktop file with this utility, it gets added with the correct version, if he just copies it manually, it doesn't.

Reply:
Actually you could use syncthing's own logging at default location for this, i.e. `Exec=/usr/bin/syncthing -no-browser -logfile=""`. In any case I think the `-no-restart` should be dropped as there is no daemon management.